<launch>
    <group ns="camera" >
        <rosparam file="$(find brobot_cam)/config/cam_config.yaml" />
        <node name="realsense" pkg="brobot_cam" type="realsense" output="screen" />

        <group ns="image_color" >
          <rosparam param="disable_pub_plugins">
            - 'image_transport/compressedDepth'
            - 'image_transport/theora'
          </rosparam>
        </group>

        <group ns="depth_map" >
          <rosparam param="disable_pub_plugins">
            - 'image_transport/compressedDepth'
            - 'image_transport/theora'
          </rosparam>
        </group>

        <group ns="depth" >
          <rosparam param="disable_pub_plugins">
            - 'image_transport/compressedDepth'
            - 'image_transport/compressed'
            - 'image_transport/theora'
          </rosparam>
        </group>

        <group ns="depth_raw" >
          <rosparam param="disable_pub_plugins">
            - 'image_transport/compressedDepth'
            - 'image_transport/compressed'
            - 'image_transport/theora'
          </rosparam>
        </group>
    </group>
</launch>
